account_mtd_vat/models/mtd_vat_endpoint.py

```diff
@@ -406,15 +406,6 @@ def _handle_vat_submit_returns_endpoint(self):
                 "Chart of Taxes can not be generated!\n " +
                 "Please create HMRC Posting Templae record first "
             )
-        for rec in hmrc_posting_config:
-            if not rec.output_account.reconcile:
-                raise exceptions.Warning(
-                    "The account {} is not marked as reconciliable !".format(rec.output_account.name)
-                )
-            elif not rec.input_account.reconcile:
-                raise exceptions.Warning(
-                    "The account {} is not marked as reconciliable !".format(rec.output_account.name)
-                )
 
         vrn = self.get_vrn(self.vrn)
         period_key = urllib.parse.quote_plus(self.select_vat_obligation.period_key)
```

Comment:
This line was removed in [PR](https://github.com/OpusVL/odoo-account-extras/pull/64), but not here. Is that specific to the Odoo version, or should it be removed here too?